content/posts/2020-07-15-tweeting-from-pull-requests/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Tweeting from Pull Requests
+author: Gustavo Barbosa
+twitter: gustavocsb
+layout: post
+lang: en
+path: /blog/tweeting-from-pull-requests
+date: '2020-07-18'
+comments: true
+---
+
+Together with [launching](https://engineering.loadsmart.com/blog/hello-world/) our blog, we have recently created (or better: _reactivated_) our Engineering team’s Twitter account. We are planning to use it as the default broadcaster for the work we do at Loadsmart. And although we may have different channels (as this blog itself is one of them) and possibly different forms of content, Twitter is great at notifying what’s new, regardless of form or platform.
+
+However, one of the biggest concerns I had when we started the idea of sharing our work was to not let our account be managed by a single person or a small group in the company. Having specific people responsible for creating those tweets would likely create bottlenecks that could lead to lack of engagement in generating new content. And to make this initiative successful, we need to make sure content creation comes from everyone. It should be accessible to anyone willing to contribute to it. I believe this is what creates a collective sense of ownership.
```

Comment:
1) "started the idea" is awkward. Maybe: One of the biggest concerns I had when we settled on the idea...
2) I'd end the sentence after contribute. Take out "to it."